test/Feature/StructuredBuffer/dec_counter.test

```diff
@@ -0,0 +1,48 @@
+#--- source.hlsl
+RWStructuredBuffer<int> Out : register(u0);
```

Comment:
>int [](http://example.com/codeflow?start=19&length=3)

Should this be uint to match the output type of DecrementCounter?

Reply:
Since we are testing the output bit pattern it doesn't really seem to matter but I went ahead and changed it